railties/lib/rails/generators/rails/app/templates/bin/yarn

```diff
@@ -0,0 +1,2 @@
+VENDOR_PATH = File.expand_path('../vendor', __dir__)
+Dir.chdir(VENDOR_PATH) { exec "yarnpkg #{ARGV.join(" ")}" }
```

Comment:
Booting a whole ruby interpreter just to shell out to yarn seems unnecessary. This is especially painful for jruby folks.